File: Programmers/Lv.1/정수 제곱근 판별/12934.java
class Solution {
    public long solution(long n) {
        long answer = -1;
        
        double num1 = Math.sqrt(n);
        long num2 = (long) num1;
        
        if (num1 == num2) {
            answer = (num2 + 1) * (num2 + 1);
        }
        
        return answer;
    }
}

File: Programmers/Lv.1/정수 제곱근 판별/12934.js
function solution(n) {
    let answer = -1;
    
    const num1 = Math.sqrt(n);
    const num2 = parseInt(num1);
    
    if (num1 === num2) {
        answer = (num2 + 1) ** 2;
    }
    
    return answer;
}
